src/components/HomeScrollingText.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
```

Comment:
```suggestion
import { useEffect } from 'react';
```
`import React from 'react';` pas utile